From ee96acef85ab60fabe95720e47ba6bf9dbac38ca Mon Sep 17 00:00:00 2001
From: jianye0918 <jian.ye@intel.com>
Date: Wed, 10 Sep 2025 12:26:47 +0800
Subject: [PATCH 12/12] Remove PTL AV1 444 Caps

PTL AV1 444 is not commited in Linux. Remove it

Backport https://github.com/intel/media-driver/commit/7086142bf96456807b3fb16d577f7fec88d98f37

Signed-off-by: Hoe, Sheng Yang <sheng.yang.hoe@intel.com>
---
 .../ddi/capstable_data_xe3_lpm_r0_specific.h  |  9 ---
 ...able_data_av1_encode_xe3_lpm_r0_specific.h | 75 -------------------
 2 files changed, 84 deletions(-)

diff --git a/media_softlet/linux/xe3_lpm_r0/ddi/capstable_data_xe3_lpm_r0_specific.h b/media_softlet/linux/xe3_lpm_r0/ddi/capstable_data_xe3_lpm_r0_specific.h
index 99d4760a8..d39063425 100644
--- a/media_softlet/linux/xe3_lpm_r0/ddi/capstable_data_xe3_lpm_r0_specific.h
+++ b/media_softlet/linux/xe3_lpm_r0/ddi/capstable_data_xe3_lpm_r0_specific.h
@@ -297,13 +297,6 @@ static const EntrypointMap entrypointMap_VAProfileAV1Profile0_Xe3_Lpm_r0
 #endif
 };
 
-static const EntrypointMap entrypointMap_VAProfileAV1Profile1_Xe3_Lpm_r0
-{
-#if defined(_AV1_ENCODE_VDENC_SUPPORTED)
-    {VAEntrypointEncSlice, &entrypointMap_VAProfileAV1Profile1_Data_Xe3_Lpm_r0},
-#endif
-};
-
 static const EntrypointMap entrypointMap_VAProfileVP9Profile0_Xe3_Lpm_r0
 {
 #if defined(_VP9_ENCODE_VDENC_SUPPORTED)
@@ -423,7 +416,6 @@ static const ProfileMap profileMap_Xe3_Lpm_r0
 
 #if defined(_AV1_ENCODE_VDENC_SUPPORTED) || defined(_AV1_DECODE_SUPPORTED)
     {VAProfileAV1Profile0, &entrypointMap_VAProfileAV1Profile0_Xe3_Lpm_r0},
-    {VAProfileAV1Profile1, &entrypointMap_VAProfileAV1Profile1_Xe3_Lpm_r0},
 #endif
 
 #if defined(_VP9_ENCODE_VDENC_SUPPORTED) || defined(_VP9_DECODE_SUPPORTED)
@@ -486,7 +478,6 @@ static const ProfileMap profileMap_Xe3_Lpm_ext_r0
 
 #if defined(_AV1_ENCODE_VDENC_SUPPORTED) || defined(_AV1_DECODE_SUPPORTED)
     {VAProfileAV1Profile0, &entrypointMap_VAProfileAV1Profile0_Xe3_Lpm_r0},
-    {VAProfileAV1Profile1, &entrypointMap_VAProfileAV1Profile1_Xe3_Lpm_r0},
 #endif
 
 #if defined(_VP9_ENCODE_VDENC_SUPPORTED) || defined(_VP9_DECODE_SUPPORTED)
diff --git a/media_softlet/linux/xe3_lpm_r0/encode/av1/ddi/capstable_data_av1_encode_xe3_lpm_r0_specific.h b/media_softlet/linux/xe3_lpm_r0/encode/av1/ddi/capstable_data_av1_encode_xe3_lpm_r0_specific.h
index 179741b52..0714aa929 100644
--- a/media_softlet/linux/xe3_lpm_r0/encode/av1/ddi/capstable_data_av1_encode_xe3_lpm_r0_specific.h
+++ b/media_softlet/linux/xe3_lpm_r0/encode/av1/ddi/capstable_data_av1_encode_xe3_lpm_r0_specific.h
@@ -105,79 +105,4 @@ static const EntrypointData entrypointMap_VAProfileAV1Profile0_Data_Xe3_Lpm_r0
     &surfaceAttribInfo_VAProfileAV1Profile0_VAEntrypointEncSlice_Xe3_Lpm_r0
 };
 
-
-static const VAConfigAttribValEncAV1Ext2 VAProfileAV1Profile1_VAEntrypointEncSlice_encAV1Ext2_Xe3_Lpm_r0
-{
-    {TILE_SIZE_BYTES - 1,3,AV1_TX_MODE_SELECT_SUPPORTED,511,0}
-};
-
-static const VAConfigAttribValEncAV1 VAProfileAV1Profile1_VAEntrypointEncSlice_encAV1_Xe3_Lpm_r0
-{
-    {0,0,0,0,0,0,1,0,0,0,0,0,1,1,0}
-};
-
-static const VAConfigAttribValEncAV1Ext1 VAProfileAV1Profile1_VAEntrypointEncSlice_encAV1Ext1_Xe3_Lpm_r0
-{
-    {31,32,1,0}
-};
-
-static ConfigDataList configDataList_VAProfileAV1Profile1_VAEntrypointEncSlice_Xe3_Lpm_r0 =
-{ 
-  {VA_RC_CQP, 0}, 
-  {VA_RC_CBR, 0}, 
-  {VA_RC_VBR, 0},
-  {VA_RC_ICQ, 0}
-}; 
-
-//!
-//! \brief  Definition for AttribList
-//!
-static const AttribList attribList_VAProfileAV1Profile1_VAEntrypointEncSlice_Xe3_Lpm_r0
-{
-   {VAConfigAttribRTFormat, VA_RT_FORMAT_YUV444 | VA_RT_FORMAT_YUV444_10},
-   {VAConfigAttribEncDynamicScaling, 0},
-   {VAConfigAttribEncDirtyRect, VA_ATTRIB_NOT_SUPPORTED},
-   {VAConfigAttribEncAV1Ext2, VAProfileAV1Profile1_VAEntrypointEncSlice_encAV1Ext2_Xe3_Lpm_r0.value},
-   {VAConfigAttribEncAV1, VAProfileAV1Profile1_VAEntrypointEncSlice_encAV1_Xe3_Lpm_r0.value},
-   {VAConfigAttribMaxPictureWidth, CODEC_8K_MAX_PIC_WIDTH},
-   {VAConfigAttribMaxPictureHeight, CODEC_8K_MAX_PIC_HEIGHT},
-   {VAConfigAttribEncAV1Ext1, VAProfileAV1Profile1_VAEntrypointEncSlice_encAV1Ext1_Xe3_Lpm_r0.value},
-   {VAConfigAttribRateControl, VA_RC_CQP | VA_RC_CBR | VA_RC_VBR | VA_RC_ICQ},
-   {VAConfigAttribEncTileSupport, 1},
-   {VAConfigAttribEncMaxRefFrames, CODEC_AV1_NUM_REFL0P_FRAMES | CODEC_AV1_NUM_REFL1B_FRAMES<<16},
-   {VAConfigAttribEncQualityRange, NUM_TARGET_USAGE_MODES - 1},
-   {VAConfigAttribEncPackedHeaders, VA_ENC_PACKED_HEADER_PICTURE | VA_ENC_PACKED_HEADER_SEQUENCE | VA_ENC_PACKED_HEADER_RAW_DATA | VA_ENC_PACKED_HEADER_MISC},
-   {VAConfigAttribEncInterlaced, VA_ENC_INTERLACED_NONE},
-   {VAConfigAttribEncQuantization, 0},
-   {VAConfigAttribEncIntraRefresh, 0},
-   {VAConfigAttribEncSkipFrame, 0},
-   {VAConfigAttribProcessingRate, VA_PROCESSING_RATE_ENCODE},
-   {VAConfigAttribEncParallelRateControl, 0},
-   {VAConfigAttribFEIMVPredictors, 0},
-};
-
-//!
-static ProfileSurfaceAttribInfo surfaceAttribInfo_VAProfileAV1Profile1_VAEntrypointEncSlice_Xe3_Lpm_r0 =
-{ 
-  {VASurfaceAttribPixelFormat, VA_SURFACE_ATTRIB_GETTABLE | VA_SURFACE_ATTRIB_SETTABLE, {VAGenericValueTypeInteger, {VA_FOURCC_AYUV}}},
-  {VASurfaceAttribPixelFormat, VA_SURFACE_ATTRIB_GETTABLE | VA_SURFACE_ATTRIB_SETTABLE, {VAGenericValueTypeInteger, {VA_FOURCC_Y410}}}, 
-  {VASurfaceAttribPixelFormat, VA_SURFACE_ATTRIB_GETTABLE | VA_SURFACE_ATTRIB_SETTABLE, {VAGenericValueTypeInteger, {VA_FOURCC_RGBA}}}, 
-  {VASurfaceAttribPixelFormat, VA_SURFACE_ATTRIB_GETTABLE | VA_SURFACE_ATTRIB_SETTABLE, {VAGenericValueTypeInteger, {VA_FOURCC_ARGB}}},
-  {VASurfaceAttribMaxWidth, VA_SURFACE_ATTRIB_GETTABLE, {VAGenericValueTypeInteger, {8192}}}, 
-  {VASurfaceAttribMaxHeight, VA_SURFACE_ATTRIB_GETTABLE, {VAGenericValueTypeInteger, {8192}}}, 
-  {VASurfaceAttribMinWidth, VA_SURFACE_ATTRIB_GETTABLE, {VAGenericValueTypeInteger, {128}}}, 
-  {VASurfaceAttribMinHeight, VA_SURFACE_ATTRIB_GETTABLE, {VAGenericValueTypeInteger, {96}}}, 
-  {VASurfaceAttribMemoryType, VA_SURFACE_ATTRIB_GETTABLE | VA_SURFACE_ATTRIB_SETTABLE, {VAGenericValueTypeInteger, {VA_SURFACE_ATTRIB_MEM_TYPE_VA | VA_SURFACE_ATTRIB_MEM_TYPE_DRM_PRIME_2}}} 
-}; 
-
-//!
-//! \brief  Definition for EmtrypointMap
-//!
-static const EntrypointData entrypointMap_VAProfileAV1Profile1_Data_Xe3_Lpm_r0
-{
-    &attribList_VAProfileAV1Profile1_VAEntrypointEncSlice_Xe3_Lpm_r0,
-    &configDataList_VAProfileAV1Profile1_VAEntrypointEncSlice_Xe3_Lpm_r0,
-    &surfaceAttribInfo_VAProfileAV1Profile1_VAEntrypointEncSlice_Xe3_Lpm_r0
-};
-
 #endif
-- 
2.48.1

